cmd/filter/osdiskexcludefilter.go

```diff
@@ -111,5 +115,5 @@ func (odf *oSDiskExcludeFilter) Include(d *controller.DiskInfo) bool {
 
 // Exclude returns true if disk devpath matches with excludeDevPath
 func (odf *oSDiskExcludeFilter) Exclude(d *controller.DiskInfo) bool {
-	return odf.excludeDevPath != d.Path
+	return !strings.HasPrefix(d.Path, odf.excludeDevPath)
```

Comment:
If the OS disk is /dev/sda, will this also exclude /dev/sda[a-z]?

Reply:
Yes, it will exclude. Haven't thought through that case of having more than 26 disks. Think we should match `/dev/sdx[numbers]`

Reply:
As more devices come available, they use different naming schemes, so perhaps a regex is better. The regex (or regexes) can be constructed with the prefix.

| disk name | partition name example |
|---|---|
| sda | sda2 |
| loop0 | loop0p1 |
| nvme3n0 | nvme3n0p2 |

Reply:
alternatively, sysfs can be searched for partitions to exclude. Glob on /sys/block/DISK_NAME/DISK_NAME*

Reply:
I think regex will be better, and keeps the logic simple. Because, extra calls will have to be introduced for searching in `sysfs`

Reply:
+1